Makefile

```diff
@@ -26,4 +26,15 @@ clean:
 	git clean -df chalice/chalicelib daemons/*/domovoilib
 	git checkout {chalice,daemons/*}/.chalice/{config,deployed}.json
 
+requirements.txt requirements-dev.txt : %.txt : %.txt.in
+	[ ! -e .requirements-env ] || exit 1
+	echo "# You should not edit this file directly.  Instead, you should edit $<." > $@
+	virtualenv --clear .requirements-env
```

Comment:
Yeah, that's true.  I prefer having L30 rather than `--clear`.  Don't want to blow anyone's env away. :)